### DISCUSSION TITLE
which is good YARN or NPM?

### DISCUSSION THREAD

**Discussion by elliotmarks** [elliotmarks/MyReactApp#2] [Q&A] [answered]
###  which is good YARN or NPM?

**Comment by jigarvakil** [comment, marked as answer]
@elliotmarks  
Yarn installs dependency packages in parallel, whereas NPM installs them sequentially. As a result, Yarn outperforms NPM when installing bigger files. Both tools can save dependent files to the offline cache.
you can refer this link: https://www.imaginarycloud.com/blog/npm-vs-yarn-which-is-better/#:~:text=As%20previously%20stated%2C%20Yarn%20installs,files%20to%20the%20offline%20cache.

**Comment by elliotmarks** [comment]
thanks @jigarvakil
